src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String[] headers = {"supply", "buy", "result"};
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method is already incorrect - it does too many things

you should have separate methods for reading data, calculating report and writing data